File: conditionals/time.py
# Douglas London, Time python
import time

#first instance of time in programming
first_time = time.gmtime()
#print(first_time)

#current time in seconds
current = time.time()
#print(curent) #seconds since 1970

#Current date and time like we see it

now = time.ctime(current)
#print(now)

#getting part of time

local_time = time.localtime(current)
day = local_time.tm_wday
hour = local_time.tm_hour # millitary time (0-23)
print(hour)

File: conditionals/time.c
// Douglas London, Time C
#include <stdio.h>
#include <time.h>




int main(void){
    //time scince Jan 1 1970
    time_t seconds;
    seconds = time(NULL);
    //printf("seconds since Jan 1, 1970 = %d\n", seconds);

    //Current time
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    //printf("current time and date is %s\n", asctime(timeinfo));

    //current hour
    time_t now = time(NULL);

    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);




    return 0;
}